app/views/RoomsListView/ServerDropdown.js

```diff
@@ -97,6 +98,15 @@ class ServerDropdown extends Component {
 		}).start(() => toggleServerDropdown());
 	};
 
+	createWorkspace = async () => {
+		logEvent(events.RL_CREATE_NEW_WORKSPACE);
+		try {
+			await Linking.openURL('https://cloud.rocket.chat/trial');
+		} catch {
+			logEvent(events.RL_CREATE_NEW_WORKSPACE_F);
```

Comment:
If there's an issue happening, log the error of the issue.

Reply:
Just remove it and keep `log(e)` only.
`logEvent` isn't important here, although you might seen it on some other catches (they are also wrong).
The only important tracking here is the error itself, so we can fix it later.